src/shared/components/theme.tsx

```diff
@@ -0,0 +1,36 @@
+import { User } from 'lemmy-js-client';
+import { Helmet } from 'inferno-helmet';
+
+export const Theme = (props: { user: User | undefined }) => {
```

Comment:
No closure, use the component style like the other components. And define the props object like the others, with user as optional.

Reply:
I changed to component style.

Optional and allowing undefined are distinctly different. Optional means you don't have to pass the prop at all. Non-optional but allowing undefined means we have to pass the prop, but the value can be undefined. In this case, this latter version fits better, because we need the user object, but we know it can sometimes be undefined.

Reply:
Gotcha.